### PR TITLE
Naming conventions altered for typescript code

### PR DESCRIPTION
**Pull request by gleb-urvanov**
Applied @Lezek123  suggestion from https://github.com/Joystream/joystream/issues/970

### CODE REVIEW

File: devops/eslint-config/index.js [hunk 1]
```diff
@@ -43,6 +43,36 @@ module.exports = {
     // should prefer using 'debug' package at least to allow control of
     // output verbosity if logging to console.
     'no-console': 'off',
+    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/class-name-casing': 'off',
+    "@typescript-eslint/naming-convention": [
+      "error",
+      {
+        selector: 'default',
+        format: ['camelCase'],
```
Comment (mnaamani) [inline devops/eslint-config/index.js:52]: I'm trying to understand the naming-conventions rules:
Is the format array listing what is allowed or considered and error?

**Comment by mnaamani** [comment]
I presume since these new rules are in https://github.com/Joystream/joystream/pull/1001 it works as you want. But why did we make an exception for the types library? Can't we have a set of rules that works for the types library and the consuming apps?

**Comment by mnaamani** [comment]
The changes are introducing some linter errors in storage node:

```
/Users/mokhtar/joystream/joystream/storage-node/packages/colossus/bin/cli.js
   13:7  error  Variable name _ must match one of the following formats: camelCase, UPPER_CASE, PascalCase  @typescript-eslint/naming-convention
  131:5  error  Method name resolve_content_id must match one of the following formats: camelCase           @typescript-eslint/naming-convention

/Users/mokhtar/joystream/joystream/storage-node/packages/colossus/lib/middleware/validate_responses.js
  45:17  error  Parameter name _ must match one of the following formats: camelCase  @typescript-eslint/naming-convention

/Users/mokhtar/joystream/joystream/storage-node/packages/discovery/discover.js
  182:16  error  Function name _discover must match one of the following formats: camelCase  @typescript-eslint/naming-convention

/Users/mokhtar/joystream/joystream/storage-node/packages/storage/storage.js
   35:7  error  Variable name _ must match one of the following formats: camelCase, UPPER_CASE, PascalCase  @typescript-eslint/naming-convention
   94:3  error  Method name _transform must match one of the following formats: camelCase                   @typescript-eslint/naming-convention
  121:3  error  Method name _flush must match one of the following formats: camelCase                       @typescript-eslint/naming-convention
  211:3  error  Method name _init must match one of the following formats: camelCase                        @typescript-eslint/naming-convention

/Users/mokhtar/joystream/joystream/storage-node/packages/storage/test/storage.js
  66:5  error  Method name resolve_content_id must match one of the following formats: camelCase  @typescript-eslint/naming-convention

/Users/mokhtar/joystream/joystream/storage-node/packages/util/lru.js
  87:3  error  Method name _prune must match one of the following formats: camelCase  @typescript-eslint/naming-convention
```

Let me add add a CI check for the storage node, and do some fixes there before we merge these new rules.

File: devops/eslint-config/index.js (hunk 1 above)
Comment (Lezek123) [reply devops/eslint-config/index.js:52]: Those are allowed

**Pull request merged**